config.js

```diff
@@ -56,8 +56,16 @@ config.access = {
     /**
      * Path of the authentication action.
      */
-    path: '/pdp/v3'
-}
+    path: '/pdp/v3',
```

Comment:
Should this PR include documentaiton of the new feature? 

At least the .md explaining the configuration parameters (as config.js has been touched to include new items), but probably in other places explainin the new accounting functionality.

Reply:
In addition, an entry in CNR file should be include